src/taipy/core/config/data_node_config.py

```diff
@@ -468,20 +479,79 @@ def _configure_sql(
     ):
         """Configure a new SQL data node configuration.
```

Comment:
Please update the description for both `_configure_sql_table` and `_configure_sql_table`, the docstring should at least gives a hint what's the difference between the 2 functions, what they're based on for example